### ISSUE TITLE
Make "TiddlyFox Apocalypse" link less prominent

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#3323]
At the moment https://tiddlywiki.com/#TiddlyFox%20Apocalypse is a very prominient link. 

We have different solutions now, which imo should be promoted instead.

**Comment by AnthonyMuscio** [comment]
Please do this, I was never that happy with this in the first place but now it has passed it may make people think TiddlyWiki does not work on firefox.I do see this in GG, newby or returning users insisting on getting their tiddlyfox fixed. Perhaps replace it with "TiddlyWiki on new Generation browsers" or something.

**Comment by TiddlyTweeter** [comment]
> Perhaps replace it with "TiddlyWiki on new Generation browsers" or something.

Actually I think even that is overkill now. Since then and before there was a huge blooming of viable  publicised solutions on saving. If anything we need a route through the maze of OPTIONS. Mat's attempt to make it easier is one route (in progress). The LESS options the better IMO. What is the minimal set needed that covers the need?

**Comment by ght** [comment]
TL;DR; Is this still an issue, with [TiddlyFox no longer available](https://addons.mozilla.org/en-GB/firefox/addon/tiddlyfox/)?

---

I would argue that now, almost 4 years later, it is not prominent enough, since TiddlyFox is no longer available AFAIU.

I went to the [Homepage](https://tiddlywiki.com/) and tried stumbling across the article in a way a new user might.

On [Getting Started](https://tiddlywiki.com/#GettingStarted), when checking only "Firefox" in the "Available methods for saving" filter, TiddlyFox is one of the first results (and the only result that says "by Jeremy Ruston", which might additionally bias new users towards it).

Opening [the result](https://tiddlywiki.com/#Saving%20with%20TiddlyFox) it is easy to miss (or dismiss and ignore) the first step, which is not styled any differently than the surrounding text and just reads:
> Ensure you have a version of Firefox before version 57. TiddlyFox will not work with Firefox 57 and on. For Firefox 57 and on, consider using the following instead:

This should link to the explanatory Apocalypse article and ideally also have a prominent warning at the top, similar to [Saving on TiddlySpot](https://tiddlywiki.com/#Saving%20on%20TiddlySpot).

**Comment by pmario** [comment]
Yes, ... That's still an issue, where the documentation can be improved.

**Comment by ght** [comment]
I will gladly help with that as soon as my understanding is more thorough. Some points I'm unsure about:

1. Is it correct that TiddlyFox is no longer usable/supported?
2. Should it still be in the search results for saving options?
3. Should the TiddlyFox tiddler still contain instructions, or just a very brief explanation of what it was and a link to "TiddlyFox Apocalypse" to explain why it no longer works, as well as links to modern alternatives? (Making the tiddler target audience previous users that want to get up to date, instead of new users that want to know how to save)

**Comment by pmario** [comment]
1. Yes. It can't be used anymore with latest browsers. .. But there are still some users out there, that use WaterFox or similar "clones", that may be able to use it. ... I'm _not_ watching that portion of the browsers, so I don't know exactly, what's going on there
2. From my opinion. No.
3. I would only add a warning and leave the content alone. It's not worth the time. The GettingStarted driver selection shouldn't show the possibility at all. ... That's just my opinion.

**Comment by TiddlyTweeter** [comment]
I last replied on this on [25 Jun 2018](https://github.com/Jermolene/TiddlyWiki5/issues/3323#issuecomment-400090731) 

First, I'm getting too old for this kind of thing.

Second I'd say the whole saving thing has gone a different direction. TidFox is irrelevant now.

@ght, what is it you need to do?

Just a query
TT

**Comment by ght** [comment]
The reason I commented here in the first place was because IMHO the high number of open issues and PRs is discouraging for potential contributors (at least for me) and I want to help get that number down, by sifting through open issues and PRs, looking for ones that can be closed/fixed with no or minimal action needed.

See how the open issue number grows here #6462, and how many issues that were deemed closable ~3 years ago are still open here #3528.

I'm not yet too accustomed to the TW community, so apologies if the ever growing number of issues is seen as a non-issue, I'll just stop then :)

**Comment by pmario** [comment]
>The reason I commented here in the first place was because IMHO the high number of open issues and PRs is discouraging for potential contributors (at least for me) and I want to help get that number down, ...

I'm with you ... I did open a new discussion here at GitHub, where I did post my thoughts, because that would derail this issue. See: https://github.com/Jermolene/TiddlyWiki5/discussions/6572

**Comment by Jermolene** [comment]
Hi @ght 

> I'm not yet too accustomed to the TW community, so apologies if the ever growing number of issues is seen as a non-issue, I'll just stop then :)

Not at all, your work is much appreciated! Like many projects, we have an embarrassing backlog of issues, and it definitely is a barrier for new users.

@pmario also makes good progress with chipping away at the backlog from time to time, and it's great to see more people willing to help out.

**Comment by Jermolene** [comment]
> I will gladly help with that as soon as my understanding is more thorough. Some points I'm unsure about:
> 
> 1. Is it correct that TiddlyFox is no longer usable/supported?

I believe that it is still possible to use it with some exotic Firefox variants. We tend to be very conservative about maintaining links to semi-obsolete stuff because we try to cater for everyone.

> 2. Should it still be in the search results for saving options?

Definitely not.

> 3. Should the TiddlyFox tiddler still contain instructions, or just a very brief explanation of what it was and a link to "TiddlyFox Apocalypse" to explain why it no longer works, as well as links to modern alternatives? (Making the tiddler target audience previous users that want to get up to date, instead of new users that want to know how to save)

I think that would make sense, yes.

**Comment by pmario** [comment]
Close this one, since the prominent link in v5.1.17 is gone
